### DISCUSSION TITLE
Incorrect values in Free Memory Pointer video

### DISCUSSION THREAD

**Discussion by Karan9034** [Cyfrin/assembly-evm-opcodes-and-formal-verification-course#19] [Q&A]
Hey @PatrickAlphaC, I think the slides in the video show incorrect value at timestamp `6:40`. Value shown in video is `0x6a` i.e. 106. 
After `0x80` i.e. 128 the next value for free memory pointer would be `0xa0`, i.e. 160 (128+32) instead.
Please correct me if I'm wrong.

**Comment by EngrPips** [comment]
He will answer this as soon as he is available.

**Reply by PatrickAlphaC** [reply]
Hi @Karan9034 thanks! What video is/was this on?

**Reply by Karan9034** [reply]
Section 2: Horse Store
Video 43: Solidity's Free Memory Pointer

**Reply by cromewar** [reply]
@Karan9034, if I'm correct, this issue was reported a long time ago, and we published an update that seems to have been reverted. 

Issue: https://github.com/Cyfrin/assembly-evm-opcodes-and-formal-verification-course/issues/6

I published an update to the platform, should be live soon, thanks for the report @Karan9034

**Reply by Karan9034** [reply]
Ah, didn't see the open issues. The videos had the incorrect values when I was watching so asked over here. Thanks for the update!
